guis/desktop/gui.py

```diff
@@ -192,11 +193,56 @@ def create_map(self):
         url = wx.FileSystem.FileNameToURL(MAP_HTML)
         self.browser.LoadURL(url)
         self.right.Add(self.browser, 1,
-                       wx.EXPAND | wx.RIGHT | wx.LEFT | wx.BOTTOM, 11)
+                wx.EXPAND | wx.RIGHT | wx.LEFT, 10)
+
+
+    def create_logical_buttons(self):
```

Comment:
Need to rename the other `create_buttons()` to something more specific now. For example, `create_select_buttons()`?